packages/theming/theme-tokens/src/getTokens.ts

```diff
@@ -0,0 +1,21 @@
+import lightAliasTokens from '@fluentui-react-native/design-tokens-windows/light/tokens-aliases.json';
+import darkAliasTokens from '@fluentui-react-native/design-tokens-windows/dark/tokens-aliases.json';
+import { hcAliasTokens } from './highContrast/tokens-alias';
+import { AppearanceOptions } from '@fluentui-react-native/theme-types';
+import { assertNever } from 'assert-never';
+
+export function getAliasTokens(mode: AppearanceOptions) {
```

Comment:
Is there a reason we can't move this file & the highContrast files over? I see adds, not a file move

Reply:
Actually, they should be moves as I didn't change the most of the files, let me try correcting this in next commit.

Reply:
I updated the change to mark this as Move, please review and let know any questions!